manifests/service/ssh.pp

```diff
@@ -6,18 +6,32 @@
 #
 # == Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
 #
-class sssd::service::ssh (
-  Optional[String]             $description             = undef,
-  Optional[Sssd::DebugLevel]   $debug_level             = undef,
-  Boolean                      $debug_timestamps        = true,
-  Boolean                      $debug_microseconds      = false,
-  Boolean                      $ssh_hash_known_hosts    = true,
-  Optional[Integer]            $ssh_known_hosts_timeout = undef
-) {
+# == Parameters available to this template:
+# debug_level
+# debug_timestampts
+# debug_microseconds
+# description
+# ssh_hash_known_hosts
+# ssh_known_hosts_timeout
+#
+
+class sssd::service::ssh {
+
   include '::sssd'
 
+  # These varaibles are referenced inside the autofs template, and
+  # because we don't want to worry about scope inside of the template
+  # we handle it here.
+
+  $description             = $sssd::description
+  $debug_level             = $sssd::debug_level
+  $debug_timestamps        = $sssd::debug_timestamps
```

Comment:
Parameters overlap with other parameters which may not have the same defaults (probably currently do, but still)